crypto/perlasm/x86_64-xlate.pl

```diff
@@ -109,14 +112,16 @@
 
 { package opcode;	# pick up opcodes
     sub re {
-	my	$self = shift;	# single instance in enough...
-	local	*line = shift;
-	undef	$ret;
+	my	$class = shift;
+	my	$self = {};
+	my	$line = shift;
```

Comment:
I would insist on keeping shifts next to each other. Alternatively one can `my ($self,$line) = @_`. It's argued that it more readable.